From a6dd12da5db40c2a235f23e6c0d2e1411c6de937 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Fri, 3 Aug 2012 16:08:07 +0800
Subject: [PATCH 1858/2463] ENGR00219360 SDMA: clean build warning

Clean build warning:
drivers/dma/imx-sdma.c: In function 'sdma_config_channel':
drivers/dma/imx-sdma.c:763: warning: unused variable 'data'

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 drivers/dma/imx-sdma.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/dma/imx-sdma.c b/drivers/dma/imx-sdma.c
index d4caaa4..e946498 100644
--- a/drivers/dma/imx-sdma.c
+++ b/drivers/dma/imx-sdma.c
@@ -760,7 +760,6 @@ static int sdma_set_chan_private_data(struct sdma_channel *sdmac)
 static int sdma_config_channel(struct sdma_channel *sdmac)
 {
 	int ret;
-	struct imx_dma_data *data = sdmac->chan.private;
 
 	sdma_disable_channel(sdmac);
 
-- 
1.7.7.4

